heron/common/tests/java/com/twitter/heron/common/config/ConfigReaderTest.java

```diff
@@ -14,27 +14,56 @@
 
 package com.twitter.heron.common.config;
 
-import java.nio.file.Paths;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
 import java.util.Map;
-import java.util.logging.Logger;
 
+import org.apache.commons.io.IOUtils;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class ConfigReaderTest {
-  private static final Logger LOG = Logger.getLogger(ConfigReaderTest.class.getName());
-
-  @Test
-  public void testLoadFile() throws Exception {
-    String file = Paths.get(System.getenv("JAVA_RUNFILES"),
-        Constants.TEST_DATA_PATH, "defaults.yaml").toString();
-    Map<String, Object> props = ConfigReader.loadFile(file);
 
+  private void testProperty(Map<String, Object> props) {
     Assert.assertEquals("role", props.get(Constants.ROLE_KEY));
     Assert.assertEquals("environ", props.get(Constants.ENVIRON_KEY));
     Assert.assertEquals("com.twitter.heron.scheduler.aurora.AuroraLauncher",
         props.get(Constants.LAUNCHER_CLASS_KEY));
-
     Assert.assertNull(props.get(Constants.USER_KEY));
   }
+
+  private InputStream loadResource() {
+    final String RESOURCE_LOC = "/heron/common/tests/resources/defaults.yaml";
+    InputStream inputStream  = ConfigReaderTest.class.
+        getResourceAsStream(RESOURCE_LOC);
+    if (inputStream == null) {
+      throw new RuntimeException("Sample output file not found");
+    }
+    return inputStream;
```

Comment:
because I also want to test loading from InputStream here: https://github.com/twitter/heron/pull/1710/files/4f721caaa991ff8c3cc156ebab9ce9b294993720#diff-535d4127d634ce5fd544311ba8e7fb2cR63